packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -207,19 +315,76 @@ class _CupertinoCheckboxState extends State<CupertinoCheckbox> with TickerProvid
   @override
   bool? get value => widget.value;
 
-  void onFocusChange(bool value) {
-    if (focused != value) {
-      focused = value;
+  WidgetStateProperty<Color> get _widgetFillColor {
```

Comment:
Should this be called `_defaultFillColor` ?

Same for the check color, side, etc, below.